
---

bool found_object
string result_path
